cluster-scope/base/core/namespaces/adoptium-project/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+- ../../../../components/project-admin-rolebindings/adoptium
```

Comment:
Quota and limit ranges are missing in here. The requester wants large quota tier.